packages/server/scripts/precheck.js

```diff
@@ -0,0 +1,25 @@
+const path = require('path');
+const { spawnSync } = require('child_process');
+const { existsSync } = require('fs');
+
+const results = [];
+
+// STEP 1 - Checks if docker-compose are available
+const dockerCheck = spawnSync('docker-compose', ['--version'])
+results.push({error: !!dockerCheck.error, message: `Docker does not seem to be installed. Please install it`});
+
+// STEP 2 - Checks if server/.env is created
+const envFile = path.resolve(__dirname, '../.env');
+results.push({error: !existsSync(envFile), message: `${envFile} file not found. Please create one, using packages/server/.env-template as base.`});
+
+let error = false;
+results.forEach((result) => {
+    if (result.error) {
+        console.log(result.message);
+        error = true;
+    }
+});
+
+if (error) {
+    throw new Error('Failed precheck');
+}
```

Comment:
To be honest this file looks a bit overkill to me. If you specify in the Readme that `docker-compose` is a prerequisite it should be enough.

Reply:
Alright, although it's not only checking for the docker-compose but also ensuring you have a .env file in it.
This kind of error could be tricky and I feel that only marking that on the README is a bit lazy, don't you think?

I could, though, make a default value for the variables, so at least we won't have errors.
What do you think?

Reply:
I did the above and changed the docs, let me know if it's cool for you